src/PowerSimulations.jl

```diff
@@ -149,7 +149,6 @@ export get_con_index
 export get_var_index
 export get_result_variable
 export get_variable_names
-export configure_logging
```

Comment:
What is the reason for the change? I thought this was desired by all. Were you getting conflicts with the IS version?